src/megachatapi_impl.cpp

```diff
@@ -3525,6 +3544,46 @@ void MegaChatCallPrivate::setError(const string &temporaryError)
     changed |= MegaChatCall::CHANGE_TYPE_TEMPORARY_ERROR;
 }
 
+void MegaChatCallPrivate::setTermCode(rtcModule::TermCode termCode)
+{
+    assert(this->termCode == rtcModule::TermCode::kNotFinished);
+    this->termCode = termCode;
+}
+
+uint8_t MegaChatCallPrivate::convertTermCode() const
+{
+    // Last four bits indicate the termination code and fifth bit indicate local or peer
+    switch (termCode & 0x0F)
```

Comment:
Fix the AND by discarding the peer flag with: `termCode & !TermCode::kPeer` (and update the comment)